.github/ISSUE_TEMPLATE/misalignment.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Content Misalignment issue
+about: Report a state when this content is misaligned with a content produced by another party.
+---
+#### Description of problem:
+
+The content is misaligned with another content that targets the same policy - typically, this means that the system hardened by our content doesn't pass the scan by the other content.
+
+
+#### Details:
+
+This content is not aligned with content from _<insert the other party name here>_
+
+The misalignment affects this content's profiles
+
+- ...
+
+The misalignment affects this content's rules
+
+- ...
+
+
+#### Outcome:
+
+- [ ] There is something to in this project to improve the content.
+  - [ ] Check needs to be improved.
+  - [ ] Remediation needs to be improved.
+- [ ] The other content's check is faulty - other party needs to be notified, they have work to do.
+
+
+#### SCAP Security Guide Version:
+
+#### The Other Policy Version:
```

Comment:
Here you probably meant the version of the other party's content implementation. Not the Policy both contents are targeting, right?
```suggestion
####  External Content's Version:
```

Reply:
Correct.